toml.md

```diff
@@ -755,13 +761,13 @@ name = "Regina Dogman"
 member_since = 1999-08-04
 ```
 
-Dotted keys define everything to the left of each dot as a table. Since tables
-cannot be defined more than once, redefining such tables using a `[table]`
-header is not allowed. Likewise, using dotted keys to redefine tables already
-defined in `[table]` form is not allowed.
-
-The `[table]` form can, however, be used to define sub-tables within tables
-defined via dotted keys.
+Dotted keys define a super-table for each name to the left of each dot that
+doesn't have a table yet. Such tables' definitions end before the next table
```

Comment:
Good advice. Didn't the spec call those "parts" before? I'll work that in.

Reply:
I need to draw a distinction between _creating_ and _defining_ tables here. If I used your text, it would imply that when a different set of dotted-key parent tables is introduced, then the previous dotted keys' tables are done being defined. They're not. We have to mention that dotted keys define their subtables, except when a subtable is already _created_.

Reply:
@ChristianSi Is this better?